src/web/templates/member/screen-o-death.html

```diff
@@ -1,5 +1,7 @@
 {% include 'base-header.html' %}
 
+<script src="//cdnjs.cloudflare.com/ajax/libs/bootbox.js/4.3.0/bootbox.min.js"></script>
```

Comment:
I've kept it here instead of base so that it only loads when status page is called. Let me know if you want to move it base-header.html